fcrepo-clients/src/test/java/edu/unc/lib/dl/fcrepo4/RepositoryObjectFactoryIT.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2016 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.dl.fcrepo4;
+
+import static com.hp.hpl.jena.rdf.model.ResourceFactory.createResource;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.net.URI;
+
+import org.fcrepo.client.FcrepoClient;
+import org.fcrepo.client.FcrepoResponse;
+import org.jgroups.util.UUID;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.hp.hpl.jena.rdf.model.Model;
+import com.hp.hpl.jena.rdf.model.ModelFactory;
+import com.hp.hpl.jena.rdf.model.Resource;
+import com.hp.hpl.jena.vocabulary.RDF;
+
+import edu.unc.lib.dl.rdf.Cdr;
+import edu.unc.lib.dl.rdf.Ldp;
+import edu.unc.lib.dl.util.RDFModelUtil;
+
+/**
+ * 
+ * @author bbpennel
+ *
+ */
```

Comment:
A little weird, but the convention for integration tests is to end the class name with IT and end unit tests with Test.  I believe this comes from the maven-surefire plugin